hugging_face_classifier_trainer/hugging_face_classifier_trainer.py

```diff
@@ -656,14 +680,9 @@ def train(
     :param random_state:            Random state for train_test_split
     """
 
-    if not label_name or not pretrained_tokenizer:
```

Comment:
You removed this yet the parameters are still optional.

Reply:
I know, there is new error if the tokenizer didn't provided, and about the labels I treat it like hugging face and set the default to be 'labels'